src/test/components/__snapshots__/Timeline.test.js.snap

```diff
@@ -3853,28 +3875,6 @@ Array [
     0.8,
     0,
   ],
-  Array [
-    "set fillStyle",
-    "#003eaa",
-  ],
-  Array [
-    "fillRect",
-    -0.4,
-    0,
-    0.8,
-    0,
-  ],
-  Array [
-    "set fillStyle",
-    "#003eaa",
-  ],
-  Array [
-    "fillRect",
-    -0.4,
-    0,
-    0.8,
-    0,
-  ],
```

Comment:
These snapshot updates only change the order of the renderings but they still render the same thing in the end.

There were two renders in here. After the refactor, now they happen in two different locations above.